.github/workflows/mainnet.yaml

```diff
@@ -0,0 +1,27 @@
+name: Node.js CI
```

Comment:
can we have different names for each type of tests (i.e., unit tests and coverage; ethereum polygon mainnet fork, polygon mainnet fork - all just suggestions) for increased readability when checking the tasks?